### PR TITLE
Address community portal issue 318 and 244

### PR DESCRIPTION
**Pull request by BradHN1**
Fixes to team creation and updating and subscribe to newsletter

### CODE REVIEW

File: src/api/handlers/team.py [hunk 1]
```diff
@@ -58,6 +58,11 @@ def create(self, request):
 
     admin_emails = args.pop('admin_emails', '')
     args["admin_emails"] = parse_str_list(admin_emails)
+
+    parentId = args.pop('parent_id', None)
+    if parentId and parentId != 'undefined':
```
Comment (massenergize-admin) [inline src/api/handlers/team.py:63]: this is great! Do you mind turning this into a helper function like `is_empty()` where it checks if its `undefined` `null` `None` etc

File: src/api/store/team.py [hunk 2]
```diff
@@ -155,12 +155,14 @@ def create_team(self, context:Context, args) -> (dict, MassEnergizeAPIError):
 
       # TODO: this code does will not make sense when there are multiple communities for the team...
       # TODO: create a rich email template for this?
-      is_published = context.user_is_admin()
+      #is_published = context.user_is_admin()
```
Comment (massenergize-admin) [inline src/api/store/team.py:158]: remove if not needed

**Review by Opoku-Agyemang — APPROVED** [review]

File: src/api/store/team.py [hunk 3]
```diff
@@ -213,12 +217,16 @@ def update_team(self, team_id, args) -> (dict, MassEnergizeAPIError):
             team.parent = parent
         
         if logo:
-          logo = Media.objects.create(file=logo, name=f"{slugify(team.name)}-TeamLogo")
-          logo.save()
-          team.logo = logo
+          # if existing logo, the string length is around 300 characters
+          # If a new logo updated, this will be the length of the file, much larger than that       
+          new_logo = len(logo) > 1000
```
Comment (Opoku-Agyemang) [inline src/api/store/team.py:222]: i am confused.  the comment talks of 300 chars and the code dictates 1000

Comment (BradHN1) [reply src/api/store/team.py:222]: That number is arbitrary.  A real image file will be at least a few kilobytes, much larger than 1000.  So 1000 seems like a safe cutoff to test for a photo upload.

**Pull request merged**